pulsar-io/kafka/src/main/java/org/apache/pulsar/io/kafka/KafkaSourceConfig.java

```diff
@@ -43,6 +43,49 @@
             "A comma-separated list of host and port pairs that are the addresses of "
           + "the Kafka brokers that a Kafka client connects to initially bootstrap itself")
     private String bootstrapServers;
+
+    @FieldDoc(
+            required = false,
+            defaultValue = "",
+            help = "Protocol used to communicate with kafka brokers.")
```

Comment:
```suggestion
            help = "Protocol used to communicate with Kafka brokers.")
```
Capitalize proper nouns correctly. Pls check all occurrences.